admin/controllers/service-manager.js

```diff
@@ -350,7 +350,7 @@ const removeCheckWindow = async (req, res, next) => {
       req.flash('error', 'Deleting an active check window is not allowed.')
     } else {
       try {
-        await checkWindowDataService.setDeletedCheckWindow(req.params.checkWindowId)
+        await checkWindowDataService.sqlSetDeletedCheckWindow(req.params.checkWindowId)
```

Comment:
`sqlDeleteCheckWindow()` might be clearer.  The implementation can still be a soft-delete.